click/button.data/button.json

```diff
@@ -1,5 +1,5 @@
 {
-    "data": [
+    "button": [
```

Comment:
Why the data name is button?

Reply:
I think you are just copy-pasting the old code without understanding. Please name things appropriately.